libs/SmartStore/src/com/salesforce/androidsdk/smartstore/store/DBHelper.java

```diff
@@ -234,9 +252,40 @@ public Cursor countQuery(SQLiteDatabase db, String table, String whereClause, St
 	 */
 	public Cursor limitRawQuery(SQLiteDatabase db, String sql, String limit, String... whereArgs) {
 		String limitSql = String.format(LIMIT_SELECT, sql, limit);
+		if (captureExplainQueryPlan) {
```

Comment:
Added the ability to turn on/off explain query plan capture.
Used by tests (to validate that the right db indexes are being used when running certain queries) but could also be used when debugging perf issues.